lib/extensions/validation_error_messages.rb

```diff
@@ -0,0 +1,5 @@
+ActiveModel::Validations.class_eval do
```

Comment:
This actually won't work - this library does NOT depend on `ActiveModel::Validations`. It's only in the test grouping of the Gemfile to show that you can bring it yourself.